packages/terra-worklist-data-grid/src/proptypes/columnHeaderCellShape.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+const columnHeaderCellShape = PropTypes.shape({
+  /**
+   * String representing a unique identifier for the column header cell.
+   */
+  id: PropTypes.string.isRequired,
```

Comment:
Do the headers need identifiers?  What would be their purpose?

Reply:
For this 1st iteration, no. But we added them as they could be helpful in event handling when advanced features such as column sorting, resizing etc. are developed.